### ISSUE TITLE
Close Effects panel Button should be Tabbable

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#8765]
### Bug description

If the User has opened the real time effects panel, the User should be able to close it via tabbing into the 'Close' button in the panel, not just 'E' shortcut. 

### Steps to reproduce

1. Open Audacity 4
2. Add a track
3. Open the Real time effect panel
4. Use Tab to navigate through the UI
5. Observe not being able to close the Real time effect panel UI

### Expected behavior

The User should be able to close the real time effect panel by tabbing to the close button. 

### Actual behavior

The User is unable to close the real time effect panel by tabbing to the close button. 

### Audacity Version

136d596817d9ea39fad9d6188d86189d8910db64

### Operating system

OS independent

### Additional context

https://github.com/user-attachments/assets/90ce1ab7-9ddf-414a-b858-833c0f230e4d

**Comment by chinakov** [comment]
This is still reproducible on the latest Master (f7034596fe9f5df9d605c52ddc4196ffee3a15b0)
